services/heroku/heroku_settings.py

```diff
@@ -74,6 +74,7 @@
 AWS_QUERYSTRING_AUTH = False
 
 # archive creation
+PHANTOMJS_BINARY = '/app/vendor/phantomjs/bin/phantomjs' # celery won't find this otherwise
```

Comment:
@jcushman I'm not thrilled about hardcoding this, but the celery worker doesn't seem to have phantomjs in its path for some reason ([see rollbar](https://rollbar.com/perma-staging/perma-staging/items/10/)) despite the fact that Django seems to find it just fine from the shell. Any ideas?